mantidproject/mantid
# All groups/pairs from home tab in muon analysis

This work allows the user to alter the default state of the multiple fitting mode in muon analysis based on the selected workspace in the home tab. 

**To test:**


1. Open muon analysis and load some data.
2. Go to the grouping tab and add a new group and a new pair. 
3. Go back to the home tab and change the selected workspace
4. Go to the settings tab and make sure multiple fitting is selected
5. Go to the data analysis tab. All groups or all pairs should be selected (depending on if you have loaded a group or pair in the home tab). The display underneath should also be correct (i.e only the groups or pairs show). 
6. Repeat 3-5 alternating between groups and pairs. 
Fixes #19712 



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
